test/testframe.c

```diff
@@ -637,7 +637,8 @@ SetTest(const char *testname, int action)
 /* Enable a test timer that will kill long-running tests, the time is configurable
  * via an environment variable.
  *
- * Only useful on POSIX systems where alarm(2) is present.
+ * Only useful on POSIX systems where alarm(2) is present. This does not include
+ * MinGW builds, which will often incorrectly decide that alarm(2) exists.
```

Comment:
Is this a side effect of needing some compilation flag for alarm to link/work correctly? I wouldn't be surprised if `AC_CHECK_FUNCS` is picking up the symbol from a header file, but it depends on something being #defined for it to actually be available.

Reply:
Apparently it's an optional thing to build in and the header and actual symbol aren't always in sync.